style..css

```diff
@@ -0,0 +1,470 @@
+body {
+    margin: 0;
+    font-family: Arial, sans-serif;
+    background-color: #f8f3f3;
+}
+
+header {
+    background-color: #f8f9fa;
+    padding: 10px 0;
+    text-align: left;
+    font: lighter;
+    color: #12a369;
+    width: fixed(1200px);
+    height: Hug(33px);
+    padding-top: 30px;
+    padding-left: 100px;
+    display: flex; 
+    align-items: center; 
+    justify-content: space-between; 
+
```

Comment:
Avoid whitespaces in your code

Reply:
Please make your code to be the same as the Figma Design
Could you please specify the areas that need correction to ensure the code matches the Figma design?

Reply:
- make the navbar same as the figma design 
- match the colors used for the header texts
- make the cards in your section 2 the same as the figma design

Reply:
Avoid making the website to overflow
Could you please explain what overflow means in this context? I’ve watched some videos but still don’t quite understand it. Also, could you clarify which part of the website experienced the overflow?

Reply:
Overflow occurs when your website displays horizontal scrollbars, which ideally should be limited to vertical scrolling. In this case, the overflow is being caused by the image or video content in the section below. To resolve this, you'll need to adjust the layout or content size to ensure it fits within the intended viewport, preventing unnecessary horizontal scrolling.

![image](https://github.com/user-attachments/assets/867e2356-49f2-443a-a656-f3f03feb443e)

if you still have any question please feel free to ask 🙌

Reply:
I understand. Thank you.

Reply:
Yes sure, adjust the font-weight to 400 it will make the text to look the same as the figma design

### And a little hint always check the properties of whether images, text, or any elements in the figma design as it is always provided. There's no need for you to be cracking your brain on the correct font size or properties to fit the original figma design

Reply:
Sir, I’ve taken advantage of everything available in Figma, but still
![image](https://github.com/user-attachments/assets/24eadbe2-2e94-4acd-b605-e8d73cb1a0c3)